graphql/schema/types/task.graphql

```diff
@@ -7,6 +7,28 @@ enum TestSortCategory {
 }
 
 ###### INPUTS ######
+"""
+TestFilterOptions is an input for the task.Tests query.
+It's used to filter, sort, and paginate test results of a task.
+"""
+input TestFilterOptions {
+  testName: String! = ""
+  statuses: [String!]
+  groupID: String! = ""
+  sort: [TestSortOptions!]
+  limit: Int! = 0
+  page: Int! = 0
```

Comment:
So I made them required because I don't want them to be pointers but I also want to enable a default value. I think that the `!` doesn't actually require that it is part of the query just that graphql will return a non-null value which will be the zero value of the type in Go. I will remove the defaults.

 @khelif96 does that make sense to you?